mithril-common/src/messages/aggregator_status.rs

```diff
@@ -70,10 +74,56 @@ mod tests {
         "total_cardano_stake": 888888888
         }"#;
 
+    #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
```

Comment:
I'm not sure that we need to do golden testing on an unreleased version of the message?

Reply:
I wonder too, since I could not decide I added as a way to discuss if it's relevant or not.